docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3.6'
```

Comment:
can we move this file under the tests or scripts dir?

Reply:
I'm thinking that we should keep it in the root because it seems like it's associated with `Dockerfile` and the recommendation is to keep that at root: https://stackoverflow.com/questions/27387811/where-to-keep-dockerfiles-in-a-project

Can you go a bit more into your reasoning here?

Reply:
This docker compose configuration stands up a Splunk Enterprise instance in Docker. It actually has nothing to do with the Splunk SDK for Python itself, it's used for testing.

I don't have a strong opinion, just wondering if that was a possibility